assets/js/modules/adsense/index.js

```diff
@@ -193,3 +200,38 @@ export const registerWidgets = ( widgets ) => {
 		[ AREA_MAIN_DASHBOARD_MONETIZATION_PRIMARY ]
 	);
 };
+
+export const ADSENSE_NOTIFICATIONS = {
+	'adsense-abr-success-notification': {
+		Component: AdBlockingRecoverySetupSuccessNotification,
+		priority: 10,
+		areaSlug: NOTIFICATION_AREAS.BANNERS_BELOW_NAV,
+		viewContexts: [ VIEW_CONTEXT_MAIN_DASHBOARD ],
+		checkRequirements: async ( { select, resolveSelect } ) => {
+			await resolveSelect( MODULES_ADSENSE ).getSettings();
```

Comment:
Since module notifications are registered regardless of the active/connected state, and settings are only preloaded for active modules, we should check that the module is connected here before loading settings, otherwise this will result in the settings being requested from the API on every page load if AdSense isn't active.